extension_config.cmake

```diff
@@ -6,4 +6,10 @@ duckdb_extension_load(iceberg
     LOAD_TESTS
 )
 
-duckdb_extension_load(tpch)
\ No newline at end of file
+duckdb_extension_load(avro
+    LOAD_TESTS
+    GIT_URL https://github.com/tishj/duckdb_avro
```

Comment:
Will change later, needed this branch (PR is submitted upstream) to detect the vcpkg-installed avro header file path

Reply:
Changed to `duckdb/duckdb_avro`